scripts/post-uninstall-rpm.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# This check works only for RPMs
+if [ $1 -ge 1 ]; then
+    # Package upgrade, do not uninstall
+    exit 0
+fi
+
+# Delete the link to the binary
```

Comment:
I adopted the POSTUN script created by electron-builder (as verified by [recreating the SPEC file](https://stackoverflow.com/questions/5613954/extract-the-spec-file-from-rpm-package)):
```bash
#!/bin/bash

# Delete the link to the binary
if type update-alternatives >/dev/null 2>&1; then
    update-alternatives --remove 'httptoolkit' '/usr/bin/httptoolkit'
else
    rm -f '/usr/bin/httptoolkit'
fi
```